persistent-mysql/ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog for persistent-mysql
 
+## (Unreleased) 2.10.4
```

Comment:
```suggestion
## 2.10.4.1
```

Reply:
It's a bugfix release, no dditions to the api, so a patch version is appropriate :)

Reply:
should that be 2.10.3.1 or I'm wrong?

Reply:
:joy: No, you're right - 2.10.3.1.

Reply:
Cool this should be ready then :)